### PR TITLE
Pivotal ID # 184199688: Add properties to starting logs

### PR DESCRIPTION
**Pull request by msrada94**
https://www.pivotaltracker.com/story/show/184199688

Please include a list of the changes:
* Logger configuration added and enable by property condition

### CODE REVIEW

File: submission/submission-webapp/src/main/resources/application.yml [hunk 1]
```diff
@@ -92,3 +92,4 @@ app:
     migration-package: ac.uk.ebi.biostd.persistence.doc.migrations
   validator:
     euToxRiskValidationApi: # EU-ToxRisk validation API
+  enablePropertiesLog: true
```
Comment (Juan-EBI) [inline submission/submission-webapp/src/main/resources/application.yml:95]: line break

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/PropertyLogger.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.event.ContextRefreshedEvent
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+
+@Component
+@ConditionalOnProperty(prefix = "app", name = ["enablePropertiesLog"], havingValue = "true")
+class PropertyLogger(@Autowired private val applicationProperties: ApplicationProperties) {
+    private val logger: Logger = LoggerFactory.getLogger(PropertyLogger::class.java)
```
Comment (Juan-EBI) [inline submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/PropertyLogger.kt:16]: this is java approach to loggin please use kotlin one. Plenty of examples in project

File: submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt [hunk 3]
```diff
@@ -14,10 +14,21 @@ class SecurityProperties {
 
     @NestedConfigurationProperty
     var instanceKeys = InstanceKeys()
+
+    override fun toString(): String {
```
Comment (Juan-EBI) [inline submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt:18]: We can transform into data class as spring support constructor binding

**Review by Juan-EBI — COMMENTED** [review]
Looking good please check comments

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/PropertyLogger.kt (hunk 2 above)
Comment (msrada94) [reply submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/PropertyLogger.kt:16]: thanks!

File: submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt (hunk 3 above)
Comment (msrada94) [reply submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt:18]: Taking a look

**Review by Juan-EBI — APPROVED** [review]
LGTM

**Pull request merged**